streamz/dataframe/core.py

```diff
@@ -3,6 +3,10 @@
 from collections import OrderedDict
 import operator
 from time import time
+try:
+    import cudf
+except ImportError:
+    cudf = None
```

Comment:
We'll need to find a place to put this.  Either a `backends.py` file within this repository, a streaming.py file in cudf, or some third tiny repository.

Reply:
I recommend the first option. Probably cudf isn't particularly interested in supporting streamz, and it seems overkill to have a whole other repo just for this.

Reply:
The current implementation does not have to import cudf from Streamz DataFrame. But I needed to modify `collection.py` file outside dataframe folder.